sdk/loadtestservice/azure-developer-loadtesting/src/samples/java/com/azure/developer/loadtesting/LongRunningOperations.java

```diff
@@ -0,0 +1,136 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.developer.loadtesting;
+
+import java.io.File;
+import java.time.Duration;
+import java.util.HashMap;
+import java.util.Map;
+
+import com.azure.core.http.rest.RequestOptions;
+import com.azure.core.util.BinaryData;
+import com.azure.core.util.polling.LongRunningOperationStatus;
+import com.azure.core.util.polling.PollResponse;
+import com.azure.core.util.polling.SyncPoller;
+import com.azure.identity.DefaultAzureCredentialBuilder;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+/**
+ * Sample demonstrates how to uploading and validating a test file, and running a test run.
+ *
+ * Authenticates with the load testing resource and shows how to uploading and validating a test file, and running
+ * a test run in a given resource.
+ *
+ * @throws ClientAuthenticationException - when the credentials have insufficient permissions for load test resource.
+ * @throws ResourceNotFoundException - when test with `testId` does not exist when uploading file.
+ */
+public final class LongRunningOperations {
+    public void beginUploadTestFile() {
+        // BEGIN: java-longRunningOperations-sample-beginUploadTestFile
+        LoadTestAdministrationClient client = new LoadTestingClientBuilder()
+                .credential(new DefaultAzureCredentialBuilder().build())
+                .endpoint("<endpoint>")
+                .buildLoadTestAdministrationClient();
+
+        String inputTestId = "12345678-1234-1234-1234-123456789abc";
+        String inputFileName = "input-test-file.jmx";
+        BinaryData fileData = BinaryData.fromFile(new File("C:/fakepath/input-file.jmx").toPath());
+        /* Note: file name passed as input argument is used, over the name in local file path */
+
+        Duration pollInterval = Duration.ofSeconds(1);
+
+        RequestOptions reqOpts = new RequestOptions()
+                .addQueryParam("fileType", "JMX_FILE");
+
+        SyncPoller<BinaryData, BinaryData> poller = client.beginUploadTestFile(inputTestId, inputFileName, fileData, reqOpts);
+        poller = poller.setPollInterval(pollInterval);
```

Comment:
I think for validation, we should just show waitForCompletion and FinalResults? Why would the user want to poll for progress on validation status, we are not spewing out logs etc. which the user might be interested in?

Reply:
We are not doing it, but if customer requires, they can add their code to print status logs. The simple cases are already covered in `HelloWorldSamples` and `ReadMeSamples`. The specific LRO files are meant for power use cases right? Should we change this to `waitForCompletion` as well